public_html/wp-content/themes/pattern-directory/src/utils/index.js

```diff
@@ -1 +1,2 @@
 export { getCategoryFromPath, removeQueryString, removeEmptyArgs } from './category';
+export { copyToClipboard } from './copy';
```

Comment:
This is a bit of a wonky file name since it's so specific but the function is kind of a one-off.

Reply:
There's a pattern in gutenberg of calling files by the name of the function they export (thinking of the `e2e-test-utils` package), so I think it's okay if you call this file `copy-to-clipboard.js` 👍🏻